google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -167,14 +174,14 @@ public Table updateTable(String datasetName, String tableName, String newFriendl
   // [TARGET delete(String, DatasetDeleteOption...)]
   // [VARIABLE "my_dataset_name"]
   public Boolean deleteDataset(String datasetName) {
-    // [START deleteDataset]
+    // [START bigquery_java_untracked]
```

Comment:
~This sample is included at https://cloud.google.com/bigquery/docs/managing-datasets#delete-dataset The Python sample has region tag `bigquery_delete_dataset`.~ Nevermind, I see this is for a method overload.